pelux.xml

```diff
@@ -71,13 +71,13 @@
   <!-- Qt + Qt Automotive support stuff -->
   <project remote="code.qt"
            upstream="5.12"
-           revision="6830447387d2bfaef1e855040068df2a7f19d485"
+           revision="b676fea3938221781bf81a8cfcf40c7d2046f8d8"
            name="yocto/meta-qt5"
            path="sources/meta-qt5"/>
 
   <project remote="code.qt"
            upstream="sumo"
-           revision="880ecb7a6e993a986d24c9a8b813bf2a48f51d24"
+           revision="5e9819044b5536eea9ad01cce0a6cc85d8b1ea6b"
```

Comment:
I would prefer to bump to 5.12 75e93a6ecb121d9aa0ee4aa986b3eef0dd20ea22 so we have a "stable" version.